usr/include/pkcs11types.h

```diff
@@ -542,6 +543,18 @@ typedef CK_ULONG CK_ATTRIBUTE_TYPE;
 #define CKA_IBM_STRUCT_PARAMS     (CKA_VENDOR_DEFINED +0x10009)
 #define CKA_IBM_STD_COMPLIANCE1   (CKA_VENDOR_DEFINED +0x1000a)
 
+/* For Dilithium, oid = 1.3.6.1.4.1.2.267.1.6.5 */
+#define IBM_DILITHIUM_KEYFORM_ROUND2        1
+
+#define CKA_IBM_DILITHIUM_KEYFORM (CKA_VENDOR_DEFINED +0xd0001)
```

Comment:
To be pedantic ;-) Please add a blank beween the '+'and '0x....'.
I see that others above also do it wrong, but lets do it right for new code.

Reply:
ok, done.